apps/api-documenter/src/yaml/IYamlTocFile.ts

```diff
@@ -34,3 +34,42 @@ export interface IYamlTocFile {
   items: IYamlTocItem[];
   metadata?: { [key: string]: string };
 }
+
+/**
+ * Typescript interface describing the config schema for toc.yml file format.
+ */
+export interface IYamlTocConfigSchema {
+  /**
+   * Represents the tree structure describing the toc.file format.
+   * Only the nodes that have an empty `items` array will be filled with API items
+   * that are matched with the filters provided. Everything else will be placed under a catchAll category
+   * that is highly recommended to be provided.
+   */
+  tocConfig: IYamlTocFile;
```

Comment:
I feel like there should be a separation between the **toc.yml** file format (as specified by DocFX) versus the **api-documenter.json** file format (which is specific to API Documenter, and may eventually be usable with non-DocFX outputs).

Reply:
How do ensure then that the TOC map user provides then is somewhat aligned with DocFX specification for a TOC